src/components/pageComponents/CustomSnackbar.js

```diff
@@ -7,19 +7,20 @@ import CloseIcon from "@material-ui/icons/Close";
 export default function CustomSnackbar(props) {
   // buttonText and onButtonClick are optional properties
   // onClose and onButtonClick should be functions
-  const { message, isOpen, onClose, buttonText, onButtonClick } = props;
+  const { buttonText, onButtonClick, ...rest } = props;
 
   return (
     <div>
       <Snackbar
+        {...rest}
         anchorOrigin={{
           vertical: "bottom",
           horizontal: "center",
         }}
-        open={isOpen}
+        // open={isOpen}
         autoHideDuration={3500}
-        onClose={onClose}
-        message={message}
+        // onClose={onClose}
+        // message={message}
```

Comment:
I know these comments should be gone. I hope its okay if we leave them for now, since the snackbar is still heavily WIP anyway

Reply:
Looks for me like the Snackbar is not working in this version as `open`, `onClose` and `message` is commented out? This would be a problem for me...

Reply:
The Snackbar has the same functionality as before. The {...rest} variable is used to inject the properties into the MaterialUI Snackbar (line 15).
This was done for convenience and to be able to use `data-testid`